README.md

```diff
@@ -102,7 +102,7 @@ Ballerina code is distributed under [Apache license 2.0](https://github.com/ball
-
+[twilio](https://github.com/ballerina-platform/module-ballerinax-twilio)|[![GitHub Release](https://img.shields.io/github/v/release/ballerina-platform/module-ballerinax-twilio?sort=semver&color=30c955&label=)](https://github.com/ballerina-platform/module-ballerinax-twilio/releases)|[![Build](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerinax-twilio/ci.yml?branch=master&label=)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/ci.yml)|[![Trivy](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerinax-twilio/trivy-scan.yml?branch=master&label=)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/trivy-scan.yml)|[![CodeCov](https://img.shields.io/badge/-N%2FA-yellow)](https://codecov.io/gh/ballerina-platform/module-ballerinax-twilio)|[![Bugs](https://img.shields.io/github/issues-search/ballerina-platform/ballerina-library?query=is%3Aopen%20label%3Amodule%2Ftwilio%20label%3AType%2FBug&label=&color=yellow)](https://github.com/ballerina-platform/ballerina-library/issues?q=is%3Aopen%20label%3Amodule%2Ftwilio%20label%3AType%2FBug)|[![GitHub Pull Requests](https://img.shields.io/github/issues-pr-raw/ballerina-platform/module-ballerinax-twilio.svg?label=)](https://github.com/ballerina-platform/module-ballerinax-twilio/pulls)|[![Load Tests](https://img.shields.io/badge/-N%2FA-yellow)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/process-load-test-result.yml)|[![GraalVM Check](https://img.shields.io/github/actions/workflow/status/ballerina-platform/module-ballerinax-twilio/build-with-bal-test-native.yml?branch=master&label=)](https://github.com/ballerina-platform/module-ballerinax-twilio/actions/workflows/build-with-bal-test-native.yml)|
```

Comment:
This is generated file, you should update modules.json file

Reply:
Changed with https://github.com/ballerina-platform/ballerina-library/pull/6066/commits/7101d40f27fb6ec31e2f1c56bdf916adf7a6800c, https://github.com/ballerina-platform/ballerina-library/pull/6066/commits/529d00f3a0f453a2be0a60a5a711582d078a97db .